### ISSUE TITLE
Prerelease updates only shows 1 relay on outlets

### ISSUE THREAD

**Issue by KyleStilkey** [jdeath/Hubspace-Homeassistant#119]
I'm not sure what's changed but since the prereleases have been pushed, my outlets lost my two relays. It's now only showing as 1 device so I can no longer individually change each one. 3.3.0 works perfectly fine. Providing a screenshot as well. 
https://github.com/user-attachments/assets/0e4676fb-55f2-4a97-a4c0-ad618b56476b

**Comment by KyleStilkey** [comment]
Side note, I also notice the wifi detail doesn't show anymore where as before the rssi gave data so that looks broken now too.

**Comment by Expl0dingBanana** [comment]
Thanks for letting me know! I saw the RSSI and figured that was a low-value one, but the other devices not being available is a bit concerning. I am currently moving a lot of the logic into the connector (in hopes to add BT), so for now I would recommend staying on 3.3.0.

**Comment by KyleStilkey** [comment]
> Thanks for letting me know! I saw the RSSI and figured that was a low-value one, but the other devices not being available is a bit concerning. I am currently moving a lot of the logic into the connector (in hopes to add BT), so for now I would recommend staying on 3.3.0.

Absolutely will and for sure, just noticed it's now function as one device vs two separate now, and as for. Me RSSI doesn't matter to me, I know some might want it, just seems like whatever the logic was used before is just broken now, you can see it's just changed itself to just the devices name but it doesn't display any data now. Just a small minor thing but some people do enjoy having this data.

**Comment by Expl0dingBanana** [comment]
@KyleStilkey If you do not need BinarySensors, can you try the new pre-release, dev-aiohubspace? This may handle the relay issue and it should solve the sensor issue (wifi-rssi).

**Comment by KyleStilkey** [comment]
> @KyleStilkey If you do not need BinarySensors, can you try the new pre-release, dev-aiohubspace? This may handle the relay issue and it should solve the sensor issue (wifi-rssi).
> 
> https://github.com/jdeath/Hubspace-Homeassistant/releases/tag/dev-aiohubspace

The Wifi sensor is working, it looks like relays are now completely gone, not even a single on or off.

![Screenshot_20250102-172735](https://github.com/user-attachments/assets/3ebe68b4-a828-4691-868c-30004b297535)

**Comment by Expl0dingBanana** [comment]
I fixed the tests for the components in the dev-aiohubspace release and reproduced the switches not mapping. This has since been resolved. Can you re-pull and try again?

**Comment by KyleStilkey** [comment]
> I fixed the tests for the components in the dev-aiohubspace release and reproduced the switches not mapping. This has since been resolved. Can you re-pull and try again?

Yes! That has fixed it! I can now see both entries again vs just 1. Thanks for the fix!

**Comment by Expl0dingBanana** [comment]
Perfect, glad it worked! I think this issue will auto-close when its merged to main (if not, ill close it at that time).

**Comment by Expl0dingBanana** [comment]
@KyleStilkey I believe the pre-release is good to go, minus final testing. Can you validate that everything still works as expected? TIA!

**Comment by KyleStilkey** [comment]
> @KyleStilkey I believe the pre-release is good to go, minus final testing. Can you validate that everything still works as expected? TIA!

Sorry I didn't respond due to life but yes it all still works and see you pushed the full release. Thanks again!

**Comment by Expl0dingBanana** [comment]
No worries! I assumed that it was probably put away for the season too, so the testing would happen next year 😂
